gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposePlugin.kt

```diff
@@ -69,8 +69,8 @@ abstract class ComposePlugin : Plugin<Project> {
         val animationGraphics get() = composeDependency("org.jetbrains.compose.animation:animation-graphics")
```

Comment:
Should be merged either to `master` or `release/1.9` depending on the discussion in https://youtrack.jetbrains.com/issue/CMP-8527/Hardcode-material3-alias-to-the-stable-version#focus=Comments-27-12340180.0-0